### DISCUSSION TITLE
S

### DISCUSSION THREAD

**Discussion by nob-rot** [ThioJoe/YT-Spam-Lists#191] [Q&A]
edit #sorry for the bad topic title#

SPAM ACCOUNTS BLOCKED BY NUM

I've been making some lists, what channel, which video, what bots are promoting the video.

I see some patterns, a lot of these video link spammers are caught with spamDomainsList, some I feel will get themselves wiped out [100+ repeat spam comments on 1 video].

The only real problem one that can't be managed currently is;
Channel puts out 4 videos a day, bots promote new video every day. It's no good blocking the video link, and they use "finally" etc type strings, so too 'false positivey' for a regex filter.

So that leaves block by account number, can that be done?
I've got these bot accounts, they all promote the one channel in the way described above, I think all of these accounts have an already blocked video link associated with them from previous closed issues.

UCdV52i7LNY04iXPAu9WgPsA | DEEPTIPS
UCG47Z7h-T1dMAh5I6xeZceg | Mama G
UC9oFyIZbbed3H7IlTVvV1Zg | Yosh Zlac
UC4_LY1b0uSReUzUX-SovTIw | Asongu Collins
UCSO6LqLhyRNg2H7_X_skY-g | NUNU Official
UCqWyMARNcIP-xVN00jDw4Ng | Reading Tips

**Comment by nob-rot** [comment]
Another "network"

UCg6rH6gUCiAh6zpjxJWVzAw
UCXKY3CSVF1rcYJVRqANZtXQ
UC4YlpcUIylflVKppA75w8Mg
UCtR3LHqBP8-RU--7G2eIvUg
UCIKjyWHmL-EQXIf_II4LWuw
UCllux1b5jKQPL-HbR7vA0Sw
UCoE74QKxsJ1wM_q9Io1aWMg
UCPagBtmM0Cd9c3glLS7eUyg
UCBfJHyJQPFy1L5EJoxSkYtQ
UCJ3GRayvB__dbUbFqCYMlEA
UC-hvEEJXi5-ztGKYG_x7TXQ
UCfGOBVHwO7z24AxY4x5eMDg
UC9vBmFEzhBJLwsW4gXBBPvA
UCypKy9D7CQyuuyievKYD_jg
UCvP7elmS4RyCUVu5U9EKoUg
UCBnQ4dZ3iSWJ4a7LtJhjKWw
UCWzHrn4fwRbrGAbO9e0miZQ
UCqEOcO1DAMQ_5lkYGa3AxVg
UCn3cf1i5MEw6S0OLNhS6Whg
UCMS0FD0VvtLFMJmIE5WHusw
UC92TG5Qx5jv8Vg9KtcLa0Xg
UCYMBsm0oQI0KSnjGqnqJ8Eg
UCJ8dfWqTS8i1Ckfo9f5KKjQ
UC_NW-LkTVU9dwWSjI17bKpA
UCVpLXQoj18Mq2UMgMrdwCZA
UCgkdXi27v4Fz-bMQnRhLd1Q
UCpO5P63kaFFL_Nr1nvS5hSA

**Comment by nob-rot** [comment]
another

UCKFb4Sfi1qiamGuBa6AmlHQ
UCjaa-mml_FPJKEBYuzNAaiw
UCpPk4lx1raZ17ycjQtIFjQA
UCrpPV1MvFV5Dsus-9omwmLw
UCSa1qxL0aRtuAG7rEhELkCw
UCqg1cnr3kkcTYtVbWw5i-Uw
UC5Qz4mJsf-1eHeaA20WasyA
UCxQg72wp_SGgGp0tpPE5TUw
UCaRQexEqj13agYXCSv1e7PQ
UCyKWbGdxqllvl4fRwdUayow
UCRDjI6_3hZN4DKgRCR_TshQ

**Comment by Firecul** [comment]
There is nothing I personally can do with this list. I'll leave it for Thio to see if it's useful in some way though.

**Comment by nob-rot** [comment]
Understood, the above accounts all have a blocked video link attached to them, but going forward they'll change video to link. Eventually they'll get new bots.
Really the channels they link to should be blacklisted, really this is YT's job, it breaks their ToS but YT doesn't care.
